drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -39,6 +39,11 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #include <net/ieee802154_radio.h>
 
+#if defined(CONFIG_SOC_NRF5340_CPUAPP) && \
+    defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
```

Comment:
Does this config imply that secure services are available? Or is any other config enabling them?

Reply:
Hm, my understanding was that it is not possible to run non-secure firmware without its secure counterpart already flashed. Gotta check.